src/checkout/checkout-service.spec.ts

```diff
@@ -294,7 +294,8 @@ describe('CheckoutService', () => {
 
             await checkoutService.loadCheckout();
 
-            expect(filter).toHaveBeenCalledWith(checkoutService.getState());
+            expect(Object.keys((filter as jest.Mock).mock.calls[0][0]))
```

Comment:
I learnt not long ago `ts-jest` provides a helper function for not having to do `as jest.Mock`:

https://github.com/kulshekhar/ts-jest/blob/master/docs/user/test-helpers.md#mocked

Reply:
Ooo, this is nice 😄